### PR TITLE
CORE-3086: Membership configuration

### PR DESCRIPTION
**Pull request by yift-r3**
Adds configuration for membership. Runtime PR in https://github.com/corda/corda-runtime-os/pull/1978

### CODE REVIEW

File: data/config-schema/src/main/resources/net/corda/schema/configuration/membership/1.0/corda.membership.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://corda.r3.com/net/corda/schema/configuration/membership/1.0/corda.membership.json",
+  "title": "Corda Membership Configuration Schema",
+  "description": "Configuration schema for the membership subsection.",
+  "type": "object",
+  "default": {},
+  "properties": {
+    "maxDurationBetweenSyncRequestsMinutes": {
+      "description": "Maximal duration in minutes between member list synchronization requests.",
```
Comment (charlieR3) [inline data/config-schema/src/main/resources/net/corda/schema/configuration/membership/1.0/corda.membership.json]:
```suggestion
      "description": "Maximum duration in minutes between member list synchronization requests.",
```

File: data/config-schema/src/main/resources/net/corda/schema/configuration/membership/1.0/corda.membership.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      "type": "integer",
+      "minimum": 10,
```
Comment (charlieR3) [inline data/config-schema/src/main/resources/net/corda/schema/configuration/membership/1.0/corda.membership.json:12]: 10 seems like a realistic minimum, but do you think someone may want to set lower for testing?

**Review by charlieR3 — DISMISSED** [review]

File: data/config-schema/src/main/resources/net/corda/schema/configuration/membership/1.0/corda.membership.json (hunk 2 above)
Comment (yift-r3) [reply data/config-schema/src/main/resources/net/corda/schema/configuration/membership/1.0/corda.membership.json:12]: I've changed the code to test it.

**Review by charlieR3 — DISMISSED** [review]

**Review by nikinagy — APPROVED** [review]
LGTM

**Review by dimosr — APPROVED** [review]

**Pull request merged**